google-cloud-firestore/src/main/java/com/google/cloud/firestore/ResourcePath.java

```diff
@@ -185,11 +185,6 @@ ResourcePath createPathWithSegments(ImmutableList<String> segments) {
   }
 
   public static Comparator<ResourcePath> comparator() {
-    return new Comparator<ResourcePath>() {
-      @Override
-      public int compare(ResourcePath left, ResourcePath right) {
-        return left.compareTo(right);
-      }
-    };
+    return Comparator.naturalOrder();
```

Comment:
just want to confirm this is tested so there's no surprises

Reply:
The code is equivalent as it ends up calling `compareTo`. The tests for this are in https://github.com/googleapis/java-firestore/blob/c5e43aacb1c8977b475b9f3eac9be6ab79c31975/google-cloud-firestore/src/test/java/com/google/cloud/firestore/OrderTest.java

Reply:
sgtm!